recipes/rhizopus/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "rhizopus" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jwergieluk/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a407e1a9d82816c6a1b116a960858a524458cb0734ac4a7a1b21ab5e6365ce8a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  commands:
+    - python --version
```

Comment:
You should at least have an import test. If that can't work for a good reason we can think of something more hacky (like the file's existence).

Reply:
Cool! I replaced this with `python -c "import rhizopus"`. I have some unit tests (which use pytest), but they are not shipped with the package to keep the list of dependencies empty.